src/code/Microsoft.PowerShell.TextUtility.csproj

```diff
@@ -17,6 +17,9 @@
         <PackageReference Include="PowerShellStandard.Library" Version="5.1.0-*">
             <PrivateAssets>all</PrivateAssets>
         </PackageReference>
+        <PackageReference Include="System.Text.Json" Version="6.0.6" >
```

Comment:
Since this module supports PS 5.1 ensure that `System.Text.Json` dll gets included in the module.